views/WelcomePageView.vue

```diff
@@ -80,11 +80,16 @@ onMounted(() => {
               <p class="text-lg font-semibold">
                 Guess the celebrity <span class="text-primary">in as few hints as possible</span>
               </p>
-              <ul class="list-disc list-inside">
-                <li>Each guess must be one of our listed <span class="text-primary">celebrities</span></li>
-                <li>Each unsuccessful guess brings a <span class="text-primary">new hint</span></li>
-                <li><span class="text-primary">Guess until you find the celebrity</span>
-                  or until <span class="text-primary">all hints are consumed</span></li>
+              <ul class="list-inside m-2">
+                <li><UIcon name="i-heroicons-arrow-long-right-16-solid"/>
+                  Each unsuccessful guess brings a <span class="text-primary">new hint</span>.</li>
+                <li><UIcon name="i-heroicons-arrow-long-right-16-solid"/>
+                  Guess until you find the <span class="text-primary">celebrity</span> or until
+                  <span class="text-primary">all hints are consumed</span>.</li>
+                <li><UIcon name="i-heroicons-arrow-long-right-16-solid"/>
+                  If you do not have the slightest <span class="text-primary">idea</span>, press the
+                  <UIcon class="text-primary" name="i-heroicons-plus-circle-16-solid"/> button to get a new hint. Be
```

Comment:
check if there is a class you can add (maybe top margin or base align) for this icon to be more aligned with the text

Reply:
unfortunately i did not find any class which works